rdtools/plotting.py

```diff
@@ -268,14 +270,65 @@ def soiling_rate_histogram(soiling_info, bins=None):
 
     soiling_summary = soiling_info['soiling_interval_summary']
     fig, ax = plt.subplots()
-    ax.hist(100.0 * soiling_summary.loc[soiling_summary['valid'], 'soiling_rate'],
-            bins=bins)
+    ax.hist(100.0 * soiling_summary.loc[soiling_summary['valid'],
+                                        'soiling_rate'], bins=bins)
     ax.set_xlabel('Soiling rate (%/day)')
     ax.set_ylabel('Count')
 
     return fig
 
 
+def tune_filter_plot(power_ac, mask, display_web_browser=False):
+    """
+    This function allows the user to visualize filtered data in
+    a Plotly plot, after tweaking the function's different
+    hyperparameters. The plot can be zoomed in on, for an in-depth look at
+    the mask in the AC power time series.
+
+    Parameters
+    ----------
+    power_ac : pd.Series
+        AC power in Watts. Index of the Pandas series is a Pandas
+        datetime index.
```

Comment:
I suggest changing the name of this parameter to be more general. For example, the plotting function works well with POA or PV energy. Maybe call it `signal` and give pv power, energy, irradiance, or temperature as examples and note that as long as they share their index with the filter, it should work.